apps/src/javalab/Backpack.jsx

```diff
@@ -0,0 +1,160 @@
+import React, {Component} from 'react';
+import Radium from 'radium';
+import color from '@cdo/apps/util/color';
+import onClickOutside from 'react-onclickoutside';
+
+const placeholderFiles = [
+  'MyClass.java',
+  'MyPainter.java',
+  'NeighborhoodMain.java'
+];
+
+const styles = {
```

Comment:
I think we are moving towards putting styles at the bottom of files, let's start using that pattern for new files